code/modules/atmospherics/components/omni_devices/_omni_extras.dm

```diff
@@ -101,16 +97,4 @@
 			return 0
 
 /proc/mode_to_gasid(var/mode)
-	switch(mode)
-		if(ATM_O2)
-			return /decl/material/gas/oxygen
-		if(ATM_N2)
-			return /decl/material/gas/nitrogen
-		if(ATM_CO2)
-			return /decl/material/gas/carbon_dioxide
-		if(ATM_N2O)
-			return /decl/material/gas/nitrous_oxide
-		if(ATM_H2)
-			return /decl/material/gas/hydrogen
-		else
-			return null
```

Comment:
It looks like this proc is useless now, at least it has the arg which unsues inside the proc.

Reply:
yeah, whoops, forgot to get rid of it.